dev_scripts/README.md

```diff
@@ -24,3 +24,18 @@ the errata_id and it submits a build in dry_run mode.
 
 By default, the script is executed on prod. To execute the script on dev the parameter `--dev` 
 should be added.
+
+
+###find_images_to_rebuild
+
+This script can be used to find images to rebuild by Errata Id Event. With the argument 
+`--container-images` can be set  list of NVRs of images to rebuild otherwise will rebuild all images
+affected by the advisory. With the argument `--cassette_path` can be set a path to the recorded 
+cassette downloaded from the Freshmaker production instance rather than querying Lightblue directly.
```

Comment:
```suggestion
affected by the advisory. The `--cassette-path` argument can be set to the path to a recorded 
cassette downloaded from the Freshmaker production. This will use the recorded Lightblue queries
instead of querying Lightblue directly.
```